notification:
  notifiers:
    robot_visualizer:
      topic: /notification_robot_state
      queue_size: 200
      latch: false
      output_devices:
        - "robot_visualizer"